community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacade.java

```diff
@@ -164,9 +163,11 @@
         <T> void unregisterTransactionEventHandler( TransactionEventHandler<T> handler );
 
         URL validateURLAccess( URL url ) throws URLAccessValidationError;
+
+        String toFacadeString();
```

Comment:
What is the purpose of this method and why it should be in `GraphDatabaseFacade`?